### ISSUE TITLE
Why is the extension "unsigned"?

### ISSUE THREAD

**Issue by jmfgdev** [jaredpar/VsVim#1983]
A day or two ago when I went to install VsVim, I noticed it was "unsigned" and decided not to install it until I ask.

I looked on the microsoft website regarding how to sign a vsix, and it said signing was beneficial to verify the integrity of the extension, if this is the case, and there's documentation, why has it not been signed?
Particularly such a popular and cool extension?

**Comment by jaredpar** [comment]
I experimented with signing in the early, early days of VsVim. I ran into a few problems with it though and eventually abandoned it. I can't remember exactly what the problem was though (that was ~7 years ago). 

Doing it now would be difficult due to how I actually create the official VSIX packages. There is a lot of unpacking, removing unnecessary files the build adds, and repacking. Can't really do that on a signed file and the tools don't support a post processing step of signing. 

> it said signing was beneficial to verify the integrity of the extension, 

I've not found this to be a problem. Sure it may be beneficial if VsVim were grabbed from an arbitrary internet location but in this case it's grabbed from the official MS owned gallery.
